addons/bestja_frontend_ucw/static/src/css/custom.css

```diff
@@ -799,4 +799,10 @@ div.carousel .carousel-indicators .active {
 .contact_us_tel_email a{
     color: #009D93;
 }
-/* End of contact page*/
\ No newline at end of file
+/* End of contact page*/
+
+/* In blog duplicate menu is needed to be hidden */
+.website_blog .dropdown-menu .duplicate{
+    display: none;
+}
+/* End of in blog duplicate menu is needed to be hidden */
```

Comment:
There should be a newline at the end of file. The red icon indicates that there is no one.